python311Packages.emoji: 2.8.0 -> 2.9.0

fabaff opened the pull request:
Diff: https://github.com/carpedm20/emoji/compare/refs/tags/v2.8.0...v2.9.0

Changelog: https://github.com/carpedm20/emoji/blob/v2.9.0/CHANGES.md

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python311Packages.emoji: 2.8.0 -> 2.9.0

Diff: https://github.com/carpedm20/emoji/compare/refs/tags/v2.8.0...v2.9.0

Changelog: https://github.com/carpedm20/emoji/blob/v2.9.0/CHANGES.md

nicoonoclaste reviewed (dismissed):
LGTM, aside from the missing `updateScript`.
Tested build on `x86_64-linux`, currently waiting on `nixpkgs-review`

fabaff commented on pkgs/development/python-modules/emoji/default.nix:
```diff
@@ -16,7 +16,7 @@ buildPythonPackage rec {
     rev = "refs/tags/v${version}";
-    hash = "sha256-fnVY4KwiqvSVYijlDckLq6qDrBJj/rJGMwaQ1mMygek=";
+    hash = "sha256-unPd0Z9hKN1mNsHleytYaKsnnMhpUn5K0ByH8vHc0AU=";
   };
```
The update was create with `nix-update`. @r-ryantm works without  `passthru.updateScript` but it can take quite some time till it pick up the latest details.

natsukium replied:
Since `updateScript` is set in python-modules, there is no need to add it explicitly unless it is complicated.
https://github.com/NixOS/nixpkgs/blob/48c81e4615ea4c451c414028434660f014d50b56/pkgs/development/interpreters/python/mk-python-derivation.nix#L308-L310

fabaff committed:
python311Packages.emoji: 2.9.0 -> 2.10.1

Diff: https://github.com/carpedm20/emoji/compare/refs/tags/v2.9.0...v2.10.1

Changelog: https://github.com/carpedm20/emoji/blob/v2.10.1/CHANGES.md

natsukium commented:
Result of `nixpkgs-review pr 273772` at 61cbcf9b run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>home-assistant-component-tests.google</li>
    <li>home-assistant-component-tests.local_calendar</li>
    <li>home-assistant-component-tests.local_todo</li>
    <li>home-assistant-component-tests.rainbird</li>
  </ul>
</details>
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/natsukium/fd1238706f2e1f4c2b1cb8e0ac1b7b38">libretranslate</a> (python311Packages.libretranslate)</li>
    <li>python311Packages.dalle-mini</li>
  </ul>
</details>
<details>
  <summary>22 packages built successfully:</summary>
  <ul>
    <li>deltachat-cursed</li>
    <li>dinghy (python311Packages.dinghy)</li>
    <li>komikku</li>
    <li>matrix-commander</li>
    <li>monophony</li>
    <li>ntfy</li>
    <li>octoprint</li>
    <li>opsdroid</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.emoji</li>
    <li>python311Packages.gcal-sync</li>
    <li>python311Packages.ical</li>
    <li>python311Packages.locationsharinglib</li>
    <li>python311Packages.mpris-server</li>
    <li>python311Packages.pyrainbird</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.translatehtml</li>
    <li>python312Packages.emoji</li>
    <li>python312Packages.mpris-server</li>
    <li>signal-export</li>
    <li>terraform-compliance</li>
  </ul>
</details>

natsukium reviewed (approved):
LGTM, thanks.

fabaff commented:
Result of `nixpkgs-review pr 273772` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages failed to build:</summary>
  <ul>
    <li>libretranslate (python311Packages.libretranslate)</li>
    <li>libretranslate.dist (python311Packages.libretranslate.dist)</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
  </ul>
</details>
<details>
  <summary>42 packages built:</summary>
  <ul>
    <li>deltachat-cursed</li>
    <li>deltachat-cursed.dist</li>
    <li>dinghy (python311Packages.dinghy)</li>
    <li>dinghy.dist (python311Packages.dinghy.dist)</li>
    <li>komikku</li>
    <li>matrix-commander</li>
    <li>matrix-commander.dist</li>
    <li>monophony</li>
    <li>ntfy</li>
    <li>ntfy.dist</li>
    <li>octoprint</li>
    <li>octoprint.dist</li>
    <li>opsdroid</li>
    <li>opsdroid.dist</li>
    <li>python311Packages.argos-translate-files</li>
    <li>python311Packages.argos-translate-files.dist</li>
    <li>python311Packages.argostranslate</li>
    <li>python311Packages.argostranslate.dist</li>
    <li>python311Packages.emoji</li>
    <li>python311Packages.emoji.dist</li>
    <li>python311Packages.gcal-sync</li>
    <li>python311Packages.gcal-sync.dist</li>
    <li>python311Packages.ical</li>
    <li>python311Packages.ical.dist</li>
    <li>python311Packages.locationsharinglib</li>
    <li>python311Packages.locationsharinglib.dist</li>
    <li>python311Packages.mpris-server</li>
    <li>python311Packages.mpris-server.dist</li>
    <li>python311Packages.pyrainbird</li>
    <li>python311Packages.pyrainbird.dist</li>
    <li>python311Packages.stanza</li>
    <li>python311Packages.stanza.dist</li>
    <li>python311Packages.translatehtml</li>
    <li>python311Packages.translatehtml.dist</li>
    <li>python312Packages.emoji</li>
    <li>python312Packages.emoji.dist</li>
    <li>python312Packages.mpris-server</li>
    <li>python312Packages.mpris-server.dist</li>
    <li>signal-export</li>
    <li>signal-export.dist</li>
    <li>terraform-compliance</li>
    <li>terraform-compliance.dist</li>
  </ul>
</details>
